wikipron/config.py

```diff
@@ -136,8 +173,7 @@ def wrapper(pron):
             # syllable boundaries. The `any()` call is much faster than
             # re.match(r"[^ˈˌ.]", pron).
             if any(c not in prosodic_markers for c in pron):
-                return pron
-
+                return combining_mods(pron) if segment else pron
```

Comment:
Shouldn't we just append `combining_mods` to the list of processors if `segment ==  True`?

Reply:
Right, that was a bit of an oversight when I added it, but now I see how simple it could be. Would it be preferable to instantiate `mods` and define the combining/modifier function directly below line 125 or would it be better to do elsewhere?

Reply:
I don't know where it should be defined exactly, honestly, myself. Should it even be defined in-class? Not sure. I guess somewhere before line 125, but where exactly I don't rightly know.

Reply:
I like the idea of keeping it partial with the processors, but the `mods` set just takes up so much space. Would it be too much to stash it somewhere and just define the combining/mods function at the top outside of the class?

Reply:
I think we need to test the new behavior in the test suite. I'd expect something similar to the following in `tests/test_config.py`:

```python
# I see we already have test_process_pron, but let's not overload it,
# and let's define a new test function just for this segmentation feature.

@pytest.mark.parametrize(
    "no_segment, input_pron, expected_pron",
    [
        # These are the test cases.
        (...),
        (...),
    ]
)
def test_no_segment(no_segment, input_pron, expected_pron):
    config = config_factory(no_segment=no_segment)
    assert config.process_pron(input_pron) == expected_pron
```